pkg/emb6/contrib/conn/udp/emb6_conn_udp.c

```diff
@@ -230,11 +230,12 @@ static void _input_callback(struct udp_socket *c, void *ptr,
 
 static void _output_callback(c_event_t c_event, p_data_t p_data)
 {
-    _send_cmd_t *send_cmd = (_send_cmd_t *)p_data;
-
     if ((c_event != EVENT_TYPE_CONN_SEND) || (p_data == NULL)) {
         return;
     }
+
+    _send_cmd_t *send_cmd = (_send_cmd_t *)p_data;
```

Comment:
I prefer to put the variable declarations (the assignment can stay here) to the top, but no-one else seems to care about this, so ACK ;-)